### ISSUE TITLE
Doesn't work for me

### ISSUE THREAD

**Issue by ndekross** [ryuran/stylus-svg#14]
Hi, 

Thanks for the plugin — this is exactly what I need. I cannot figure out how to use it though. 

```stylus
use('/node_modules/svg-stylus/index.js')

.foo
  background-image: svgImport('test.svg') 
```

compiles to

```css
.foo {
  background-image: svgImport('test.svg');
}
```

WAIDW?

Thanks.

**Comment by ndekross** [comment]
Never mind, I solved it myself, but you need to change the installation instructions in the README file. 

So, basically your instructions which I mindlessly copied:

`npm install svg-stylus`

This installs another plugin and it does basically the same. While yours plugin can be installed as 

`npm install stylus-svg`

Please, update your documentation.

Cheers!

**Comment by ryuran** [comment]
OUPS ! It still an issue I will fix it

**Comment by ryuran** [comment]
Fixed in v1.1.3
